### DISCUSSION TITLE
IntelliJ me tira Build success pero el autograding me tira Build Failure

### DISCUSSION THREAD

**Discussion by matiasyogui** [dds-jv/foro#200] [Consultas de herramientas] [answered]
Buenas, recien entregue el tp de macowins luego de haber corrido los tests con 'mvn verify', el cual me tiro los siguiente
![image](https://user-images.githubusercontent.com/48895837/116277376-cc840a00-a75b-11eb-9d6f-743caf4619a8.png)

Luego hago la entrega final con el comando que dice el repo 'mvn clean verify && git tag entrega-final && git push origin HEAD --tags', pero en el autograding me aparece Build Failure.
![image](https://user-images.githubusercontent.com/48895837/116277941-5b912200-a75c-11eb-886f-96f6024fafb7.png)

No se de donde es el problema.

**Comment by matiasyogui** [comment, marked as answer]
Supongo que el problema viene de que me olvide de commitear antes de tirar el comando 'mvn clean verify && git tag entrega-final && git push origin HEAD --tags'. No se si la entrega era unica, pero ya tengo listo para pushear jaja.

**Reply by matiasyogui** [reply]
Termine borrando el tag y pusheando de nuevo pero con el commit. Y me anduvo todo bien, los tests pasaron.

**Reply by nanderson87** [reply]
Hola,

Creo que no tanto porque lo tuvieras commiteado o no. Más porque entre un comando  y otro  hubo algún cambio que hizo que ya no compile. Ya que el error es de maven al intentar de compilar. 

Pero genial, si borraste el tag y volviste a correr el comando` 'mvn clean verify && git tag entrega-final && git push origin HEAD --tags'` y todo termino bien sería suficiente.
